infinispan-remote/cache-alias/pom.xml

```
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>infinispan-simple-tutorials-remote-cache-alias</artifactId>
+    <parent>
+        <relativePath>../../pom.xml</relativePath>
+        <version>1.0.0-SNAPSHOT</version>
+        <groupId>org.infinispan.tutorial.simple</groupId>
+        <artifactId>infinispan-simple-tutorials</artifactId>
+    </parent>
+    <name>Infinispan Simple Tutorials: Cache Alias</name>
```

niptpicking, to be consistent with the name in embedded tutorial
```suggestion
    <name>Infinispan Simple Tutorials: Remote Cache Alias</name>
```